src/packer.pkr.hcl

```diff
@@ -46,32 +46,51 @@ variable "skip_create_ami" {
   type        = bool
 }
 
-data "amazon-ami" "debian_bullseye" {
+# The 'winrm_password' variable is configured as an optional variable because
+# the cisagov/pre-commit-packer hook does not support passing variables to
+# `packer validate`. Once this limitation is removed this should be changed to
+# a required variable. Please see the following for more information:
+# https://github.com/cisagov/windows-server-packer/issues/21
+variable "winrm_password" {
+  default     = ""
+  description = "The password used to connect to the instance via WinRM."
+  sensitive   = true
+  type        = string
+}
+
+variable "winrm_username" {
+  default     = "Administrator"
+  description = "The username used to connect to the instance via WinRM."
+  type        = string
+}
+
+data "amazon-ami" "windows_server_2022" {
   filters = {
-    name                = "debian-11-amd64-*"
+    name                = "Windows_Server-2022-English-Full-Base-*"
     root-device-type    = "ebs"
     virtualization-type = "hvm"
   }
   most_recent = true
-  owners      = ["136693071363"]
+  owners      = ["amazon"]
   region      = var.build_region
 }
 
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
-source "amazon-ebs" "example" {
+source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
     device_name           = "/dev/xvda"
     encrypted             = true
     volume_size           = 8
     volume_type           = "gp3"
   }
-  ami_name                    = "example-hvm-${local.timestamp}-x86_64-ebs"
+  ami_name                    = "windows-server-2022-${local.timestamp}-x86_64-ebs"
```

Comment:
Do we want to update the AMI name?

Reply:
Yes, this repo should definitely produce an AMI with a different name so that we can differentiate it from the AMI produced by `windows-server-packer`.

Reply:
Addressed here https://github.com/cisagov/windows-commando-vm-packer/commit/fca2728ac8fdb8298991237ff2c1d66dc921be17#diff-61505b29b2bfc888d227eb40217a3c6d44ec3ba4dc88b44bdf89a7fa63d3358fR88